src/wallet/rpcwallet.cpp

```diff
@@ -1375,8 +1375,9 @@ static void MaybePushAddress(UniValue & entry, const CTxDestination &dest)
  * @param  fLong      Whether to include the JSON version of the transaction.
  * @param  ret        The UniValue into which the result is stored.
  * @param  filter     The "is mine" filter bool.
```

Comment:
IMO this should be renamed to avoid confusion, to something like `filter_ismine`